### PR TITLE
Docstring for function 1

### PR DESCRIPTION
**Pull request by kmoravej**
#8

### CODE REVIEW

File: mdspyutils/mdspyutils.py [hunk 1]
```diff
@@ -2,13 +2,18 @@
 
 
 def parse_lab(file_name: str) -> list:
-    """[summary]
+    """This function parse the content of MDS lab files
```
Comment (rtaph) [inline mdspyutils/mdspyutils.py]: "parses"

Comment (rtaph) [reply mdspyutils/mdspyutils.py]: For now, we are only giving the option to parse markdown content right? If so, I would add that to the title.

**Review by rtaph — CHANGES_REQUESTED** [review]

**Review by sukh2929 — COMMENTED** [review]
It looks good!

File: mdspyutils/mdspyutils.py (hunk 1 above)
Comment (sukh2929) [reply mdspyutils/mdspyutils.py]: 3a5efa9 resolved via Kamal.

**Review by sukh2929 — APPROVED** [review]

**Review by rtaph — APPROVED** [review]

**Pull request merged**